.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
+  "root": true // needed to disable parent eslint
+}
```

Comment:
This make changes very annoying, because I change whole file in result, when do small change.
I think I also saw other PRs that also had this problem.

I always use linking + `packages/` folder. So project take eslint config from root project.
It is better to configure something within library, but at least we can avoid changes for whole file, because everyone have different config right now.

Reply:
Got it, awesome.